package.json

```diff
@@ -3,6 +3,7 @@
   "version": "0.0.0",
   "license": "MIT",
   "source": "src/index.ts",
+  "type": "module",
```

Comment:
@jivey this is the only way I was able to use es6 imports in the script but it causes the following error in linting:
```
Error [ERR_REQUIRE_ESM]: require() of ES Module /home/runner/work/assignment-components/assignment-components/.eslintrc.js from /home/runner/work/assignment-components/assignment-components/node_modules/@eslint/eslintrc/dist/eslintrc.cjs not supported.
.eslintrc.js is treated as an ES module file as it is a .js file whose nearest parent package.json contains "type": "module" which declares all .js files in that package scope as ES modules.
Instead rename .eslintrc.js to end in .cjs, change the requiring code to use dynamic import() which is available in all CommonJS modules, or change "type": "module" to "type": "commonjs" in /home/runner/work/assignment-components/assignment-components/package.json to treat all .js files as CommonJS (using .mjs for all ES modules instead).
```
still looking into solutions.